modules/publicize/publicize.php

```diff
@@ -418,6 +419,91 @@ function save_meta( $post_id, $post ) {
 		// Next up will be ::publicize_post()
 	}
 
+	public function update_published_message( $messages ) {
+		global $post_type, $post_type_object, $post;
+		if ( ! $this->post_type_is_publicizeable( $post_type ) ) {
+			return $messages;
+		}
+		$view_post_link_html = '';
+		$viewable = is_post_type_viewable( $post_type_object );
+		if ( $viewable ) {
+			$view_text = __( 'View post' ); // intentinally omitted domain
+
+			if ( $post_type == 'jetpack-portfolio' ) {
+				$view_text = esc_html__( 'View project', 'jetpack' );
+			}
+
+			$view_post_link_html = sprintf( ' <a href="%1$s">%2$s</a>',
+				esc_url( get_permalink( $post ) ),
+				$view_text
+			);
+		}
+
+		$services = $this->get_publicizing_services( $post->ID );
+		if( empty( $services ) ) {
+			return $messages;
+		}
+
+		$labels = array();
+		foreach( $services as $service ) {
+			$labels[] = sprintf(
+				_x( '%1$s: %2$s', 'Service: Account publicizing to', 'jetpack' ),
+				esc_html( $service['service_name'] ),
+				esc_html(  $service['display_name'] )
+			);
+		}
+
+		$messages['post'][6] = sprintf(
+			_x( 'Post published and publicizing to %1$s. %2$s', 'Published Message: Accounts to publish to, link to view published post', 'jetpack' ),
+			implode(',', $labels ),
+			$view_post_link_html );
+
+		if ( $post_type == 'post' && class_exists('Jetpack_Subscriptions' ) ) {
+			$subscription = Jetpack_Subscriptions::init();
+			if ( $subscription->should_email_post_to_subscribers( $post ) ) {
+				$messages['post'][6] = sprintf(
+					_x( 'Post published, sending emails to subscribers and publicizing to %1$s. %2$s', 'Published Message: Accounts to publish to, link to view published post', 'jetpack' ),
+					implode(',', $labels ),
+					$view_post_link_html );
+			}
+		}
+
+		$messages['jetpack-portfolio'][6] = sprintf(
+			_x( 'Project published and publicizing to %1$s. %2$s', 'Published Message: Accounts to publish to, link to view published post', 'jetpack' ),
+			implode(',', $labels ),
+			$view_post_link_html );
+
+		return $messages;
+	}
+
+	function get_publicizing_services( $post_id ) {
+		$services = array();
+
+		foreach ( (array) $this->get_services( 'connected' ) as $service_name => $connections ) {
+			// services have multiple connections.
+			foreach ( $connections as $connection ) {
+				$unique_id = '';
+				if ( !empty( $connection->unique_id ) )
```

Comment:
Can we use braces here, as per https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style